tests/dragonfly/acl_family_test.py

```diff
@@ -457,6 +457,23 @@ async def test_require_pass(df_factory):
     await client.close()
 
 
+@pytest.mark.asyncio
+@dfly_args({"port": 1111, "requirepass": "temp"})
+async def test_require_pass_with_acl_file_order(df_factory, tmp_dir):
+    acl = create_temp_file(
+        "USER default ON >jordan ~* +@all",
+        tmp_dir,
+    )
+
+    df = df_factory.create(aclfile=acl)
+    df.start()
+
+    client = aioredis.Redis(username="default", password="jordan", port=df.port)
+
+    b"OK" == await client.execute_command("SET foo bar")
```

Comment:
that's not an assert 🤔 Also why not use set instead of execute_command, it returns a bool

Reply:
LOL I had an assert over there :rofl: